### DISCUSSION TITLE
Queuing Tracks for Live Play (and other ideas, discussion)

### DISCUSSION THREAD

**Discussion by maxwellsmart84** [SynthstromAudible/DelugeFirmware#463] [Ideas]
It would be really awesome if there was a way to queue a track before it's played the main out through the headphone port.  I think this would make the deluge such a better instrument for live performance play.  Would love to hear other ideas on this (and know if this is even possible with software). Maybe another state besides "mute" on the track that is "preview".  This would give it some of the functionality of a DJ mixer

Thanks!

**Comment by jamiefaye** [comment]
The headphone jack on the Deluge uses the same signal pathway as the main outputs, so we can't do what you want to do with the hardware we have.

However there are ways to make cueing easier on the Deluge. Several ideas include: using Downrush (which runs from a special type of SD-card with WiFi) to preview samples; Using two Deluges with a DJ mixer; compiling a special version of the Deluge Firmware that runs on a smart phone or tablet, creating a special build of the Deluge Firmware which splits processing into 2 mono channels, and ideas we haven't thought of yet.

**Comment by PaulFreund** [comment]
Unfortunately the current hardware is not able to drive the headphone and main output separately but I agree it would be a great Feature!

**Comment by ChamCy** [comment]
Hi everyone. Hope you're doing good. I agree, the possibility of cueing would be a massive improvement for the live or DJ setup. This possibility and Ableton link..  So, I've got a question for you. Would it be possible to trick a little and use the L/R out to solution this cueing problem? One jack L output for the master out and R for cueing? This way you also ear both in the headphone. Maybe after cueing off, a possibility to switch back on stereo out and use all the panning functionalities..? Do you think it could be possible? Thx

https://forums.synthstrom.com/discussion/5559/queuing-a-track-or-an-instrument-for-live-set/p1?new=1
